bluemira/builders/EUDEMO/tf_coils.py

```diff
@@ -397,17 +397,36 @@ def build_xy(self) -> Component:
 
         return component
 
-    def build_xyz(self) -> Component:
+    def build_xyz(self, degree: float = 360.0) -> Component:
         """
         Build the x-y-z components of the TF coils.
+
+        Parameters
+        ----------
+        degree: float
+            The angle [°] around which to build the components, by default 360.0.
+
+        Returns
+        -------
+        component: Component
+            The component grouping the results in 3D (xyz).
         """
         component = Component("xyz")
 
+        # Minimum angle per TF coil (nudged by a tiny length since we start counting a
+        # sector at theta=0). This means we can draw a sector as 360 / n_TF and get one
+        # TF coil per sector. Python represents floats with 16 significant figures before
+        # getting round off, so adding on 1e-13 works here, in case someone sets n_TF
+        # to be 2.
+        min_tf_deg = (360.0 / self._params.n_TF.value) + 1e-13
+        n_tf_draw = min(int(degree // min_tf_deg) + 1, self._params.n_TF.value)
+        degree = min_tf_deg * n_tf_draw
+
```

Comment:
I wasn't so sure what the expected behaviour should be here. I went with making sure that one coil is shown per sector, which I think would generally be what people would expect to happen, but in that building through 180 degrees will not give the coil opposite the first (which I guess could be another expected way that this would work).

Reply:
It looks "right" to me.